lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -52,17 +52,22 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
             rewriter.getF64FloatAttr(beta));
 
         // Expression: alpha * x + beta
-        Value alpha_x_plus_beta = rewriter.create<Torch::AtenAddScalarOp>(
-            binder.getLoc(), resultType, tensorOperand, constBeta,
-            /*alpha=*/constAlpha);
+        Value alphaMulX = rewriter.create<Torch::AtenMulScalarOp>(
+            binder.getLoc(), resultType, tensorOperand, constAlpha);
+        Value constOne = rewriter.create<Torch::ConstantFloatOp>(
+            binder.getLoc(), rewriter.getType<Torch::FloatType>(),
+            rewriter.getF64FloatAttr(1.0));
+        Value alphaMulXPlusBeta = rewriter.create<Torch::AtenAddScalarOp>(
+            binder.getLoc(), resultType, alphaMulX, constBeta,
+            /*alpha=*/constOne);
```

Comment:
AtenAddScalarOp has the input signature (tensor, float, float) and performs result = operand[0] + operand[1]*operand[2], correct? I feel like this op would be way more useful if it multiplied the tensor operand[0] with operand[2].

Reply:
i wonder if this is just to be consistent with the aten.add.Tensor op

where you get tensor1+tensor2*alpha

Reply:
but also this leave the question of why we have the alpha in the first place ... for quantization maybe?